.github/renovate.json

```diff
@@ -6,5 +6,36 @@
   "separateMajorMinor": false,
   "groupName": "all",
   "schedule": "before 6am every weekday",
-  "ignoreDeps": ["github-scm-filter-aged-refs", "azure-vm-agents"]
-}
+  "packageRules": [
+    {
+      "description": "Confirm that latest version doesn't cause errors and remove this by 10/07/2025. See: https://github.com/hmcts/cnp-jenkins-docker/pull/1524",
+      "matchManagers": [
+        "maven"
+      ],
+      "matchPackageNames": [
+        "github-scm-filter-aged-refs"
+      ],
+      "automerge": false
+    },
+    {
+      "description": "Confirm that this works with Ubuntu 24.04 and remove this by 10/07/2025. See: https://github.com/jenkinsci/azure-vm-agents-plugin/pull/603",
+      "matchManagers": [
+        "maven"
+      ],
+      "matchPackageNames": [
+        "azure-vm-agents"
+      ],
+      "automerge": false
+    },
+    {
+      "description": "Should be removed once https://www.jenkins.io/security/advisory/2020-04-07/#SECURITY-1633 has been resolved.",
+      "matchManagers": [
+        "maven"
+      ],
+      "matchPackageNames": [
+        "gatling"
```

Comment:
tbh why not just put a version range on this, so that any versions after `136.vb_9009b_3d33a_e` will work?

The Jenkins security team have confirmed its fixed on the master branch

Reply:
Updated now. Is that along the lines of what you're thinking?